abci/example/kvstore/kvstore_test.go

```diff
@@ -70,6 +70,24 @@ func TestKVStoreKV(t *testing.T) {
 	testKVStore(t, kvstore, tx, key, value)
 }
 
+func TestPersistentKVStoreEmptyTX(t *testing.T) {
+	dir, err := os.MkdirTemp("/tmp", "abci-kvstore-test") // TODO
+	if err != nil {
+		t.Fatal(err)
+	}
+	kvstore := NewPersistentKVStoreApplication(dir)
+	tx := []byte("")
+	reqCheck := types.RequestCheckTx{Tx: tx, Type: types.CheckTxType_New}
```

Comment:
```suggestion
	reqCheck := types.RequestCheckTx{Tx: tx}
```

Reply:
`CheckTxType_New` is the default, so no need to fill it up.
However, up to you if you want to keep it for readability